x/clp/keeper/rewards.go

```diff
@@ -56,15 +56,20 @@ func (k Keeper) DistributeDepthRewards(ctx sdk.Context, blockDistribution sdk.Ui
 		return err
 	}
 
+	partitions, err := k.GetAllLiquidityProvidersPartitions(ctx)
```

Comment:
@sifag I am not sure about the compute time of this one, but if it is heavy could be worth calling it only if `shouldDistribute` is one what do you think?

Reply:
Yep, just pushed